core/src/apps/common/paths.py

```
@@ -29,7 +29,7 @@ class KeychainValidatorType(Protocol):
         def is_in_keychain(self, path: Bip32Path) -> bool:
             ...
 
-        def verify_path(self, path: Bip32Path, force_strict: bool = False) -> None:
+        def verify_path(self, path: Bip32Path, force_strict: bool = True) -> None:
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)